content/news/draft-bevy-0.14/index.md

```diff
@@ -20,6 +20,11 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 - **Computed states and substates:** modeling complex app state is a breeze with these type-safe extensions to our `States` abstraction.
 - **Rounded corners:** rounding off one of `bevy_ui`'s roughest edges, you can now procedurally set the corner radius on your UI elements.
 
+For the first time, Bevy 0.14 was prepared using a **release candidate** process to help ensure that you can upgrade right away with peace of mind right away.
```

Comment:
Is the double "right away" intentional? Might read better with just one.

Reply:
```suggestion
For the first time, Bevy 0.14 was prepared using a **release candidate** process to help ensure that you can upgrade right away with peace of mind.
```